website/management/commands/create_file_upload_tasks.py

```
@@ -0,0 +1,237 @@
+from django.core.management.base import BaseCommand
+from website.models import Labs, TaskContent, Tasks
+
+
+class Command(BaseCommand):
+    help = "Creates File Upload Vulnerabilities lab tasks with theory and simulation content"
+
+    def handle(self, *args, **kwargs):
```

Ruff (0.14.3)

Unused method argument: `args`

(ARG002)